<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>760</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>40</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>房贷计算器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
border-image: url(:/C:/Users/dell/Desktop/QQ图片20200527165520.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;贷款类别：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="currentText">
     <string>商业贷款</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>商业贷款</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>公积金贷款</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>组合型贷款</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;计算方式：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>根据面积、单价计算</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>根据贷款总额计算</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;房屋单价：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string extracomment="                                                      ">请输入房屋单价                               </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;房屋面积：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入房屋面积</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;元/平米&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;平方米&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;按揭成数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>8成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5成</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2成</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;按揭年数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>25年（300期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24年（288期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23年（276期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22年（264期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21年（252期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20年（240期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19年（228期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18年（216期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17年（204期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16年（192期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15年（180期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14年（168期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13年（156期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12年（144期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11年（132期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10年（120期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9年（108期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8年（96期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7年（84期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6年（72期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5年（60期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4年（48期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3年（36期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2年（24期）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1年（12期）</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>等额本金</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>等额本息</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;还款方式：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Arial&quot;;
font: 75 18pt &quot;Bahnschrift&quot;;
font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>610</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>10</horstretch>
      <verstretch>4</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Arial&quot;;
font: 87 18pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>清空重填</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;贷款利率：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入贷款利率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>68</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>qingkong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>828</x>
     <y>728</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>jisuan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lineeditchanged()</slot>
  <slot>qingkong()</slot>
  <slot>jisuan()</slot>
 </slots>
</ui>
